daemon/src/db.rs

```diff
@@ -716,6 +695,22 @@ mod tests {
         assert_eq!(data.len(), 100);
     }
 
+    #[tokio::test]
+    async fn inserting_two_cfds_with_same_order_id_should_fail() {
+        let mut conn = setup_test_db().await;
+
+        let cfd = Cfd::dummy().insert(&mut conn).await;
```

Comment:
I think you could literally just call `insert` twice here.

Reply:
This would panic because `insert` uses `unwrap` inside.